src/content/troubleshooting/_index.en.md

```diff
@@ -117,38 +117,44 @@ If the status indicates any other error, run `kubectl -n submariner-operator get
 
-If the Multiclusterservice resource was created properly on the cluster, run `kubectl -n submariner-operator describe multiclusterservice <your-multiclusterservice-name>` and check if it has the correct `ClusterID` and `ServiceIP`:
+If the ServiceImport resource was created properly on the cluster, run `kubectl -n submariner-operator describe serviceimport <your-serviceimport-name>` and check if it has the correct `ClusterID` and `ServiceIP`:
 
 ```
-Name:         nginx-demo-default-cluster2
+Name:         nginx-default-cluster2
 Namespace:    submariner-operator
-Labels:       submariner-io/clusterID=cluster2
-Annotations:  <none>
-API Version:  lighthouse.submariner.io/v1
-Kind:         MultiClusterService
+Labels:       <none>
+Annotations:  origin-name: nginx
+              origin-namespace: default
+API Version:  lighthouse.submariner.io/v2alpha1
+Kind:         ServiceImport
 Metadata:
-  Creation Timestamp:  2020-04-16T15:40:36Z
+  Creation Timestamp:  2020-07-14T17:27:32Z
   Generation:          1
-  Resource Version:    6847
-  Self Link:           /apis/lighthouse.submariner.io/v1/namespaces/submariner-operator/multiclusterservices/nginx-demo-ns-default-src-cluster2
-  UID:                 9db7759c-7ff8-11ea-85a3-0242ac110006
+  Resource Version:    2790
+  Self Link:           /apis/lighthouse.submariner.io/v2alpha1/namespaces/submariner-operator/serviceimports/nginx-default-cluster2
+  UID:                 4cbe1c2b-c5f7-11ea-9bbe-0242ac110009
 Spec:
-  Cluster Service Info:
-    Cluster Domain:  
-    Cluster ID:      cluster2       ==========> ClusterID of cluster where service is running
-    Service IP:      100.92.243.156 ==========> ServiceIP or GlobalIP of service you're trying to access
-Events:              <none>
+  Ports:                    <nil>
+  Session Affinity:
+  Session Affinity Config:  <nil>
+  Type:                     SuperclusterIP
+Status:
+  Clusters:
+    Cluster:  cluster2 ==========> ClusterID of cluster where service is running
+    Ips:
+      100.92.43.63     ==========> ServiceIP or GlobalIP of service you're trying to access
+Events:  <none>
 ```
 
-If the data is not correct, you can manually edit the Multiclusterservice resource to set the correct IP as a workaround and create an [issue](https://github.com/submariner-io/lighthouse/issues) with relevant information.
+If the data is not correct, you can manually edit the ServiceImport resource to set the correct IP as a workaround and create an [issue](https://github.com/submariner-io/lighthouse/issues) with relevant information.
 
 If it is correct, congratulations - you've found a new [issue](https://github.com/submariner-io/lighthouse/issues).
```

Comment:
I don't understand this last sentence -- should we just remove it?

Reply:
It mentions you can use a workaround but still it is better to raise an issue, I think the concern is about  this one "If it is correct, congratulations - you've found a new [issue]"

Reply:
Yes. I am only talking about the last part. I mean, if it's successful you should go have a beer and not report an issue ;)

Reply:
:) ... No, the workaround(manually adding the IP, which LH should have added) was required since there is a bug in LH, so that needs to be reported before having a beer. ;)

Reply:
Oh ok, makes sense I misinterpreted it we can remove it.

Reply:
Thanks